src/auth/entities/user.entity.ts

```
@@ -97,9 +99,14 @@ export class User {
     example: '2023-01-01T00:00:00.000Z',
   })
   @CreateDateColumn()
-  createdAt: Date;
+  createdAt: Date
```

ESLint

[error] Insert `;`

(prettier/prettier)